pkg/core/function.go

```diff
@@ -167,6 +150,25 @@ func (c *client) CreateFunction(options CreateFunctionOptions, log io.Writer) (*
 	return s, nil
 }
 
+func (c *client) writeRiffToml(options CreateFunctionOptions) error {
```

Comment:
@ericbottard What's the point of writing a `riff.toml` file? I couldn't see that in the code that was deleted, so is this a new feature?

Reply:
That's the means the riff buildpack [uses](https://github.com/projectriff/riff-buildpack/blob/ecff5d07810a7d6d15f10164c4e1ea3c5b73917e/metadata.go#L50) to get some of its info (propagated [here](https://github.com/projectriff/riff-buildpack-group/blob/9d744887a7a9175d478e6c309fbd8bacc1453101/riff-cnb-buildtemplate.yaml#L48) when going the BuildTemplate way)

Reply:
Or to phrase it otherwise, yes this is a "new feature":
> Adds support for latest BP build for --local-path

Reply:
Once there is a standard way to pass env values to the build pack we should use that instead of riff.toml,  it for now we should consider it an implementation detail.

Reply:
Sounds ok. Can this overwrite a riff.toml provided by the user?

Reply:
Yes, but that should never be a problem (what they pass via the CLI should be the source of truth. As @scothis stated, this should really be seen as an implementation detail IMHO)

Reply:
The only problem is overwriting the user's file. Wouldn't it be better to fail and require the user to delete it?

Reply:
I really don't think so. I think that slows down the workflow. There should not be a "user's file" in the first place anyway